pkg/util/metrics_helper_test.go

```diff
@@ -770,7 +771,7 @@ func TestUserRegistries_RemoveUserRegistry_HardRemoval(t *testing.T) {
 	mainRegistry := prometheus.NewPedanticRegistry()
 	mainRegistry.MustRegister(tm)
 
-	// Soft-remove single registry.
+	// Hard-remove single registry.
```

Comment:
This looks unrelated to the rest of the PR? Is it just a random comment fix, or an accident?

Reply:
This is fix for copy&pasted comment. `true` parameter on next line indicates "hard" removal.